addons/website/tests/test_snippets.py

```diff
@@ -77,3 +77,13 @@ def test_10_parallax(self):
 
     def test_11_snippet_images_wall(self):
         self.start_tour('/', 'snippet_images_wall', login='admin')
+
+    def test_12_snippet_popup_and_scrollbar(self):
+        website = self.env.ref('website.default_website')
+        website.cookies_bar = True
+        self.start_tour(self.env['website'].get_client_action_url('/'), 'snippet_popup_and_scrollbar', login='admin')
+
+    def test_12_snippet_popup_and_animations(self):
```

Comment:
12 12

I think we should stop using this notation, so either use `13` or remove the numbers
I was in favor of using it for a long time but the only value using it is to spot if someone remove a test silently for convenience which we are not ok with.
But with code owner, this is unlikely we miss it.

Reply:
> 12 12
> 
> I think we should stop using this notation, so either use `13` or remove the numbers I was in favor of using it for a long time but the only value using it is to spot if someone remove a test silently for convenience which we are not ok with. But with code owner, this is unlikely we miss it.

ok, I removed the number.. I'll let you confirm that we no longer use them

Reply:
I confirm, but it's not that big of a deal is someone keep using it